mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/massindexing/MassIndexer.java

```diff
@@ -143,6 +143,19 @@ public interface MassIndexer {
 	 */
 	MassIndexer failureHandler(MassIndexingFailureHandler failureHandler);
 
+	/**
+	 * Sets the maximum number of failures that will be reported and sent to {@link MassIndexingFailureHandler} per indexing operation.
+	 * Any failures exceeding this number will be ignored. A count of such ignored failures together with the operation they belong to
+	 * will be logged upon the completion of indexing process.
```

Comment:
It's not very clear what you mean by "indexing operation". Maybe use "indexed type" instead? That's not strictly true, but at least the meaning is clearer.